docs/trace/getting-started/README.md

```diff
@@ -75,7 +52,7 @@ Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/maste
 The activitysource instance is used to start an `Activity` which represent
```

Comment:
I think it makes sense to link `ActivitySource` and `Activity` to the Microsoft docs just to clearly draw people to the relationship.

Reply:
I did not do it intentionally as the official microsoft docs don't yet exist. The link to ActivitySource in API doc also gives the wrong impression that its only available for .NET 5 only.
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.activitysource?view=net-5.0